layerstats/README.md

```diff
@@ -0,0 +1,175 @@
+Layer Stats
+===========
+
+This page describes how to generate and analyze layer stats data to find ways to optimize tile size.
+
+### Generating Layer Stats
+
+Run planetiler with `--output-layerstats` to generate an extra `<output>.layerstats.tsv.gz` file with a row per
+tile layer that can be used to analyze tile sizes. You can also
+get stats for an existing archive by
+running:
+
+```bash
+java -jar planetiler.jar stats --input=<path to mbtiles or pmtiles file> --output=layerstats.tsv.gz
+```
+
+The output is a gzipped tsv with a row per layer on each tile and the following columns:
+
+|       column        |                                                                   description                                                                   |
+|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| z                   | tile zoom                                                                                                                                       |
+| x                   | tile x                                                                                                                                          |
+| y                   | tile y                                                                                                                                          |
+| hilbert             | tile hilbert ID (defines [pmtiles](https://protomaps.com/docs/pmtiles) order)                                                                   |
```

Comment:
should this be the first column, indicating that everything is sorted by this? (is that true for mbtiles?)

Reply:
If you run this over an mbtiles file, it will be sorted by z/x/y. I was kind of leaning towards z/x/y first since those are the more human-readable tile coordinates?

Reply:
Right, that makes sense then.